api/webhook.go

```diff
@@ -23,6 +26,12 @@ func InitWebhook(r *mux.Router) {
 	sr.Handle("/outgoing/regen_token", ApiUserRequired(regenOutgoingHookToken)).Methods("POST")
 	sr.Handle("/outgoing/delete", ApiUserRequired(deleteOutgoingHook)).Methods("POST")
 	sr.Handle("/outgoing/list", ApiUserRequired(getOutgoingHooks)).Methods("GET")
+
+	sr.Handle("/{id:[A-Za-z0-9]+}", ApiAppHandler(incomingWebhook)).Methods("POST")
```

Comment:
@hmhealey No difference between the two routes. (you can see that they have the same handler) We want to switch to the new route, but want to keep supporting the old one for awhile. 
Documentation should be updated to the new route. Ticket: https://mattermost.atlassian.net/browse/PLT-2402